builds/misc/templates/build-packages.yaml

```diff
@@ -3,6 +3,9 @@ parameters:
     type: boolean
     default: false
 
+variables:
+  DisableDockerDetector: true
```

Comment:
Adding this variable to a template doesn't really help (it's redundant), but also doesn't hurt.

Reply:
Actually, it looks like adding the variable to the template _is_ a problem:
![image](https://user-images.githubusercontent.com/9042469/171741441-358a4402-7d58-459b-84d9-e93e956b7263.png)